graphdatascience/query_runner/arrow_query_runner.py

```diff
@@ -52,12 +52,24 @@ def warn_about_deprecation(self, old_endpoint: str, new_endpoint: str) -> None:
             DeprecationWarning(f"The endpoint '{old_endpoint}' is deprecated. Please use '{new_endpoint}' instead.")
         )
 
-    def run_query(
+    def run_cypher(
         self,
         query: str,
         params: Optional[Dict[str, Any]] = None,
         database: Optional[str] = None,
         custom_error: bool = True,
+    ) -> DataFrame:
+        return self._fallback_query_runner.run_cypher(query, params, database, custom_error)
+
+    def call_endpoint(
+        self,
+        type: EndpointType,
+        endpoint: str,
+        yields: Optional[List[str]] = None,
+        body: Optional[str] = None,
```

Comment:
I wonder if `body` is a good name. It's not really a body in the conventional sense of a function body, or if statement body. It looks more like `param_placeholders` or something to me

Reply:
I was also not hapy with this. At first i head `positional_args`. thinking about it more though, it is indeed in most cases a 1:1 mapping with the keys in params 🤔 
Should add a check if thats correct